README.md

```diff
@@ -388,7 +404,7 @@ Options:
                         Password for specified user. Will prompt if omitted
   --fqdn=FQDN           Set an explicit FQDN, overriding detected FQDN from
                         socket.getfqdn(), currently detected as
-                        client.example.com
+                        bs-1234.example.org
```

Comment:
Yeah, I saw that after sending the PR last night. Airplane wifi was crap so I couldn't amend.